openshift/patches/remove_resource_version_check.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/vendor/knative.dev/eventing/pkg/scheduler/statefulset/scheduler.go b/vendor/knative.dev/eventing/pkg/scheduler/statefulset/scheduler.go
+index a95242ee2..083767450 100644
+--- a/vendor/knative.dev/eventing/pkg/scheduler/statefulset/scheduler.go
```

Comment:
wanna include the autoscaler log patch too? Just wondering

Reply:
Ah right

Reply:
Yes, thanks!